hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -1593,7 +1595,12 @@ public Map<byte[], List<HStoreFile>> close() throws IOException {
   public static final long DEFAULT_CLOSE_WAIT_INTERVAL = 10000; // 10 seconds
 
   public Map<byte[], List<HStoreFile>> close(boolean abort) throws IOException {
-    return close(abort, false);
+    return close(abort, false, false);
+  }
+
+  public Map<byte[], List<HStoreFile>> close(boolean abort, boolean isGracefulStop)
```

Comment:
I think there's some confusion with the parameters here. We should keep the original signature `public Map<byte[], List<HStoreFile>> close(boolean abort, boolean ignoreStatus)` and pass `ignoreStatus` along in the delegating call (here we are always forcing it to false). Could it be behind the latest UT failureS?

Reply:
@wchevreuil I have addressed this feedback. However, the previous UTs are probably not failing with this change here. Let's wait for the pre-commit to see how it goes this time.